### PR TITLE
r-spatstat

### PR DESCRIPTION
**Pull request by jsignell**

### CODE REVIEW

File: recipes/r-spatstat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    - r-matrix
+    - r-abind
+    - r-deldir >=0.0_21
+    - r-goftest
+    - r-mgcv
+    - r-nlme
+    - r-polyclip >=1.5_0
+    - r-rpart
+    - r-spatstat.data >=1.1_0
+    - r-spatstat.utils >=1.7_1
```
Comment (jdblischak) [inline recipes/r-spatstat/meta.yaml]: r-spatstat.data and r-spatstat.utils need to be added to conda-forge before this recipe can be built. @jsignell could you please send separate PRs for these two recipes?

**Comment by jdblischak** [comment]
@jsignell In addition to submitting recipes for the dependencies, please update your recipe using https://github.com/bgruening/conda_r_skeleton_helper to add more maintainers and add the GPL license.

**Comment by cbrueffer** [comment]
Added in #4849
